### PR TITLE
Capsule Cert Generate Absolute path w/o

### PR DESCRIPTION
**Pull request by jyejare**
w/o for bug https://bugzilla.redhat.com/show_bug.cgi?id=1466688

### CODE REVIEW

File: automation_tools/satellite6/capsule.py [hunk 1]
```diff
@@ -206,7 +207,11 @@ def generate_capsule_certs(capsule_hostname, force=False):
         fqdn_opt = '--foreman-proxy-fqdn'
     else:
         fqdn_opt = '--capsule-fqdn'
-    cert_path = '{0}-certs.tar'.format(capsule_hostname)
+    if bz_bug_is_open(1466688):
+        # Absolute path bug
+        cert_path = '~/{0}-certs.tar'.format(capsule_hostname)
```
Comment (sghai) [inline automation_tools/satellite6/capsule.py]: this may not work because of another bug: 1470653

Comment (jyejare) [reply automation_tools/satellite6/capsule.py]: We are not double quoting the cert_path string so it should work

Comment (sghai) [reply automation_tools/satellite6/capsule.py]: Looks like automation won't include quotes.. in that case it will work.

**Comment by sghai** [comment]
ACK

**Pull request merged**